dietpi/func/dietpi-set_hardware

```diff
@@ -2259,6 +2259,84 @@ _EOF_
 
 	}
 
+	#/////////////////////////////////////////////////////////////////////////////////////
+	# Dedicated GPU
+	# INPUT_DEVICE_VALUE
+	# - None (Uninstall)
+	# - Intel
+	# - Nvidia
+	# - AMD
+	# - Custom
+	#/////////////////////////////////////////////////////////////////////////////////////
+	DedicatedGPU_Main(){
+	
+		#Current GPU
+		local gpu_current=$(sed -n '/^[[:blank:]]*CONFIG_GPU_DRIVER=/{s/^[^=]*=//p;q}' /boot/dietpi.txt)
+		[[ $gpu_current ]] && gpu_current=${gpu_current,,} || gpu_current='custom'
+		[[ $INPUT_DEVICE_VALUE ]] || INPUT_DEVICE_VALUE='custom'
+		
+		#Steam installed for additional drivers?
+		local steam_installed=0
+		dpkg-query -s 'steam' &> /dev/null && steam_installed=1	
+		
+		if [[ $INPUT_DEVICE_VALUE == 'custom' ]]
+		then
+		
+			echo '0' &> /dev/null #Do nothing
+		
+		elif (( $G_HW_ARCH == 10 ))
+		then		
+		
+			if [[ $INPUT_DEVICE_VALUE == 'none' || 
+				$gpu_current != $INPUT_DEVICE_VALUE ]] #Change in HW vendor, uninstall 1st.
+			then
+
+				local uninstall_array=('mesa-vulkan-drivers:*') #Mesa Vulkan
+				uninstall_array+=('i965-va-driver' 'xserver-xorg-video-intel') #Intel
+				uninstall_array+=('nvidia-driver' 'nvidia-driver-libs:*' 'libnvidia-glvkspirv:*' 'nvidia-xconfig' 'libgl1-nvidia-glx' 'nvidia-settings' 'nvidia-vulkan-common' 'nvidia-vulkan-icd:*') #Nvidia
+				uninstall_array+=('firmware-amd-graphics' 'xserver-xorg-video-amdgpu') #AMD
+				
+				apt-mark auto "${uninstall_array[@]}"
+				G_AGA
+
+			fi
+
+			#installs
+			local install_array=()		
+			if [[ $INPUT_DEVICE_VALUE == 'intel' ]]
+			then
+
+				install_array+=('i965-va-driver' 'xserver-xorg-video-intel' 'libgl1-mesa-dri' 'mesa-vulkan-drivers')
```

Comment:
Let's not install the legacy Intel DDX. All cards from 2007 and above work better with the default modesetting DDX according to package description.
```suggestion
				install_array+=('i965-va-driver' 'libgl1-mesa-dri' 'mesa-vulkan-drivers')
```